Sources/KeyboardShortcuts/Utilities.swift

```diff
@@ -87,6 +87,60 @@ final class LocalEventMonitor {
 }
 
 
+final class RunLoopLocalEventMonitor {
+	private let runLoopMode: RunLoop.Mode
+	private let callback: (NSEvent) -> NSEvent?
+	private let observer: CFRunLoopObserver
+
+	init(
+		events: NSEvent.EventTypeMask,
+		runLoopMode: RunLoop.Mode,
+		callback: @escaping (NSEvent) -> NSEvent?
+	) {
+		self.runLoopMode = runLoopMode
+		self.callback = callback
+
+		self.observer = CFRunLoopObserverCreateWithHandler(nil, CFRunLoopActivity.beforeSources.rawValue, true, 0) { _, _ in
+			var eventsToHandle = [NSEvent]()
+
+			while let eventToHandle = NSApp.nextEvent(matching: .any, until: nil, inMode: .default, dequeue: true) {
```

Comment:
I think you can just pass `events` to `matching` here instead of checking inside the loop.

Reply:
👍 It would be good to have a short code comment about that as it was definitely not clear by just reading the code.